### PR TITLE
Dcka 828 did management

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/features/didManagement/ExportDIDScreen.js [hunk 1]
```diff
@@ -38,8 +39,8 @@ export function ExportDIDScreen() {
               <Input
                 placeholder="Password"
```
Comment (maycon-mello) [inline src/features/didManagement/ExportDIDScreen.js:40]: Shouldn't we use locales here?

File: src/features/didManagement/ExportDIDScreen.js [hunk 2]
```diff
@@ -56,8 +57,8 @@ export function ExportDIDScreen() {
               <Input
                 placeholder="Confirm password"
```
Comment (maycon-mello) [inline src/features/didManagement/ExportDIDScreen.js:58]: We could use locales here

File: src/features/didManagement/ExportDIDScreen.js [hunk 3]
```diff
@@ -101,14 +102,81 @@ export function ExportDIDScreen() {
...
+  const handleChange = useCallback(
+    key => {
+      return value => {
+        const updatedForm = {
+          ...form,
+          [key]: value,
+        };
+
+        if (key === 'password') {
+          updatedForm.lengthValidation = value.length >= 8;
```
Comment (maycon-mello) [inline src/features/didManagement/ExportDIDScreen.js:132]: can we test this? maybe move this code to a separate function

File: test/testSetup.js [hunk 4]
```diff
@@ -27,7 +27,18 @@ jest.mock('@react-native-async-storage/async-storage', () => mockAsyncStorage);
...
-jest.mock('react-native-share', () => 'RNShare');
+jest.mock('react-native-share', () => {
+  const open = jest.fn(() => Promise.resolve([]));
+  return {
+    __esModule: true,
+    default: {
+      open,
+    },
+  };
+  // return {
```
Comment (maycon-mello) [inline test/testSetup.js:38]: can we remove this code?

**Review by maycon-mello — CHANGES_REQUESTED** [review]
left some comments

File: test/testSetup.js (hunk 4 above)
Comment (osinakayah) [reply test/testSetup.js:38]: It is needed for the mock to work. I wanted to mock the "open" function in the react-native-share library.

**Review by maycon-mello — APPROVED** [review]

**Pull request merged**